pkg/services/sqlstore/migrations/ualert/tokenizer_test.go

```diff
@@ -0,0 +1,216 @@
+package ualert
```

Comment:
I did not notice negative tests... for example `$instance`,`$$$$`, `${test`, `$${test}` etc